assets/sass/components/global/_googlesitekit-new-badge.scss

```diff
@@ -19,8 +19,11 @@
 .googlesitekit-new-badge {
 	background-color: $c-violet-v-500;
 	color: $c-violet-v-0;
+	font-size: $fs-title-sm;
+	line-height: $lh-title-sm;
 	margin-left: 6px;
 	text-transform: none;
+	vertical-align: baseline;
```

Comment:
I find this addition puts the text out of alignment, but it may only be with the other changes mentioned above here.

Reply:
Makes sense to revert this to middle with the padding and font size changes, now it's aligned correctly in the middle (like Figma).